src/pages/gd/Bridge/index.tsx

```diff
@@ -15,6 +15,10 @@ const Bridge = memo(() => {
     const { library } = useActiveWeb3React()
     const [theme] = useApplicationTheme()
 
+    const successHandler = useCallback((e) => {
+        console.log(e)
```

Comment:
non informative, should be something like `console.log('Kima bridge error:', e.message, e')`